verifier.sh

```diff
@@ -269,7 +269,8 @@ if [ \$GEM_SET_DEBUG ]; then
 fi
 cd ~/$GEM_GIT_PACKAGE
 virtualenv --system-site-packages platform-env
-. platform-env/bin/activate
+source platform-env/bin/activate
+pip install simplejson==2.0.9
```

Comment:
Please comment this line and maybe add a comment; the issue with simplejson has been fixed in the LXC